build-configs.yaml

```diff
@@ -607,13 +607,22 @@ build_configs:
           arm64:
             extra_configs:
               - 'allmodconfig'
+              - 'allnoconfig'
           arm:
-            base_defconfig: 'multi_v7_defconfig'
+            filters:
+              - regex: {
```

Comment:
Note that you don't actually need the `{ }` brackets when defining a dictionary on multiple lines.  But that's just cosmetics so not a blocker, and this file should be reviewed and cleaned up a bit anyway as things keep getting added to it.